WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -276,7 +255,7 @@
           <%= f.button :submit, t('.submit_create_value'), class: "btn-primary" %>
         <% end %>
 
-        <% if @competition.persisted? && !@competition_admin_view && !is_actually_confirmed && current_user.can_confirm_competition?(@competition) %>
+        <% if @competition.persisted? && !@competition_admin_view && !is_actually_confirmed && current_user.can_confirm_competition?(@competition) && @competition.events.count > 0 %>
```

Comment:
Hmmm... this feels like a duplication of this logic we already have in our Competition model: https://github.com/thewca/worldcubeassociation.org/blob/8b7e8e363606b576735aa73a90cf8815c8d1b22f/WcaOnRails/app/models/competition.rb#L99-L111.

I think it would be better to present the user with the button and when they fail to confirm, they get a good error message that makes it clear they need to add events. @jonatanklosko's suggestion to show an alert when there are currently no events might go well with this.